documents-webapp/src/main/webapp/vue-app/documents/components/body/table/DocumentInfoDrawer.vue

```diff
@@ -261,6 +268,33 @@ export default {
         }
       }));
     },
+    getDocumentView() {
+      const currentUrlSearchParams = window.location.search;
+      const queryParams = new URLSearchParams(currentUrlSearchParams);
+      if (queryParams.has('view')) {
+        const view = queryParams.get('view');
+        this.viewTab = view.toLowerCase() === 'folder' ? 'Folder' : 'RECENT';
```

Comment:
```suggestion
        this.viewTab = view.toLowerCase() === 'folder' ? 'folderView' : 'recentView';
```
Use these i18n keys or this value will not be tranalsted
https://github.com/exoplatform/documents/pull/559/files#diff-1c3c4454aaeda7c8871802fca60959bfcef929df893b1e09e66cc023359b11dbL3

Reply:
+ there is a better way to detect the view, juts pass the attribute selectedView as prop from root component to the infi drawer component : 
https://github.com/exoplatform/documents/blob/develop/documents-webapp/src/main/webapp/vue-app/documents/components/DocumentsMain.vue#L134
timeline means 'recentview'